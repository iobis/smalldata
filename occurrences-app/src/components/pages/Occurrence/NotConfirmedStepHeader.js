import classNames from 'classnames'
import PropTypes from 'prop-types'
import React from 'react'
import StepHeader from './StepHeader'

export default function NotConfirmedStepHeader({ className, onStepTitleClick, stepDescription, stepTitle }) {
  return (
    <StepHeader
      className={classNames('not-confirmed has-text-white', className || 'has-background-info')}
      dataDescription=""
      iconVisible={false}
      onStepTitleClick={onStepTitleClick}
      selectedData=""
      stepDescription={stepDescription}
      stepTitle={stepTitle}/>
  )
}

NotConfirmedStepHeader.propTypes = {
  className:        PropTypes.string,
  onStepTitleClick: PropTypes.func.isRequired,
  stepDescription:  PropTypes.string.isRequired,
  stepTitle:        PropTypes.string.isRequired
}
