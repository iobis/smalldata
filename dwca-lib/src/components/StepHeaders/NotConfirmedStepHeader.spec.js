import NotConfirmedStepHeader from './NotConfirmedStepHeader'
import React from 'react'
import renderer from 'react-test-renderer'

describe('NotConfirmedStepHeader', () => {
  it('renders correctly', () => {
    expect(renderer.create(
      <NotConfirmedStepHeader
        onStepTitleClick={() => {}}
        stepDescription="step description"
        stepTitle="step title"/>
    ).toJSON()).toMatchSnapshot()
  })
})
