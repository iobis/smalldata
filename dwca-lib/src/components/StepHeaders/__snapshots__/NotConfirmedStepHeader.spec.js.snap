// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotConfirmedStepHeader renders correctly 1`] = `
<header
  className="step-header columns is-vcentered not-confirmed has-text-white has-background-info"
  onClick={[Function]}
>
  <div
    className="column is-1 is-hidden-mobile"
  >
    <p
      className="datadescription"
    >
      
    </p>
  </div>
  <p
    className="selected-data column is-hidden-mobile"
  >
    
  </p>
  <div
    className="column details"
  >
    <h3
      className="is-size-6"
    >
      step description
    </h3>
    <h2
      className="step-title"
    >
      step title
    </h2>
  </div>
</header>
`;
